components/dashboard/src/App.tsx

```diff
@@ -37,7 +37,6 @@ const Projects = React.lazy(() => import(/* webpackPrefetch: true */ './projects
 const Project = React.lazy(() => import(/* webpackPrefetch: true */ './projects/Project'));
```

Comment:
**suggestion**: Could we add a hover state for the more actions button on the project cards? See https://github.com/gitpod-io/gitpod/issues/4704 for specs. 🙏

Reply:
let's tackle this separately.

Reply:
DEAL 🤝